site/src/component/SideInfo/SideInfo.tsx

```diff
@@ -161,6 +162,8 @@ const SideInfo: FC<SideInfoProps> = (props) => {
         </div>
       </div>
 
+      {props.terms && <RecentOfferings terms={props.terms ? props.terms : []} />}
```

Comment:
The `&&` and `props.terms ? props.terms : []` are redundant. Assuming you don't want to render the recent offerings table if there are none:
```suggestion
      {props.terms && <RecentOfferings terms={props.terms} />}
```